README.md

```diff
@@ -33,7 +33,7 @@ Building
 --------
 
 Build dependencies are systemd, cURL, and json-c, which development packages
-are `systemd-devel`, `json-c-devel`, and `libcurl-devel` on RPM-based distros,
+are `autoreconf`, `automake`, `libtool`, `rpm-build`, `systemd-devel`, `json-c-devel`, and `libcurl-devel` on RPM-based distros,
```

Comment:
We need to specify that autoreconf, automake, and libtool are needed only for building from Git, and rpm-build is only needed for building the RPM package.